src/Microsoft.Diagnostics.Monitoring.Options/DiagnosticPortOptionsDefaults.cs

```diff
@@ -7,5 +7,6 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi
     internal class DiagnosticPortOptionsDefaults
     {
         public const DiagnosticPortConnectionMode ConnectionMode = DiagnosticPortConnectionMode.Connect;
+        public const bool ClearEndpointOnStartup = false;
```

Comment:
I think we should make this `true` in 7.x by default. I'm not sure if it should be `true` for the 6.x images since that would be a breaking change (although is suspect it won't be an issue for most users).

Reply:
I think that it is fair to say that this is a bug fix and thus applicable to 6.x (and the 6.2 release as well).

Reply:
My concern here was that making this the default behavior will lead to a set of secondary issues that may be much harder to diagnose. People who are for some reason running two dotnet-monitors won't naturally fail due to binding but rather succeed and then get very bizarre behavior.

Reply:
Would a reasonable middle ground be that we set as an environment variable in the dotnet-monitor container to `true` and leave it as `false` by default in the tool?